extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -670,24 +670,27 @@ public void handle(AsyncResult<String> event) {
                 if (event.failed()) {
                     Throwable effectiveCause = event.cause();
                     if (effectiveCause instanceof BindException) {
+                        List<Integer> portsUsed = Collections.emptyList();
+                        
                         if ((sslConfig == null) && (httpServerOptions != null)) {
-                            effectiveCause = new QuarkusBindException(httpServerOptions.getPort());
+                            portsUsed = List.of(httpServerOptions.getPort());
                         } else if ((httpConfiguration.insecureRequests == InsecureRequests.DISABLED) && (sslConfig != null)) {
-                            effectiveCause = new QuarkusBindException(sslConfig.getPort());
+                            portsUsed = List.of(sslConfig.getPort());
                         } else if ((sslConfig != null) && (httpConfiguration.insecureRequests == InsecureRequests.ENABLED)
-                                && (httpServerOptions != null)) {
-                            effectiveCause = new QuarkusBindException(
-                                    List.of(httpServerOptions.getPort(), sslConfig.getPort()));
+                                   && (httpServerOptions != null)) {
+                            portsUsed = List.of(httpServerOptions.getPort(), sslConfig.getPort());
                         }
+                        
+                        effectiveCause = new QuarkusBindException((BindException)effectiveCause, portsUsed);
                     }
+                    LOGGER.error(effectiveCause.getMessage());
```

Comment:
Yeah, I think we should drop it. I have already seen this error message in the console so I think it's properly propagated. It's easy to check it though if you want, just start the server with an already used port.